tests/functional/reporting-postMessage.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+const intern = require("intern").default;
+const { assert } = intern.getPlugin("chai");
+const { registerSuite } = intern.getInterface("object");
+const FunctionalHelpers = require("./lib/helpers.js");
+
+const url = intern.config.siteRoot + "/issues/new";
+
+// This string is executed by calls to `execute()` in various tests
+// it postMessages a small green test square.
+const POSTMESSAGE_TEST =
+  'postMessage({message: {"url":"http://www.test.com","utm_source":"desktop-reporter","utm_campaign":"report-site-issue-button","src":"desktop-reporter","details":{"gfx.webrender.all":false,"gfx.webrender.blob-images":true,"gfx.webrender.enabled":false,"image.mem.shared":true,"buildID":"20191016225400","channel":"default","consoleLog":[],"hasTouchScreen":false,"mixed active content blocked":false,"mixed passive content blocked":false,"tracking content blocked":"false","hasMarfeel":true},"extra_labels":["type-marfeel"]}}, "http://localhost:5000")';
```

Comment:
`test.com` is a real domain name, please replace with `example.com` which is a reserved domain specifically for documentation, see [RFC 6761](https://tools.ietf.org/html/rfc6761#section-6.2).

Reply:
Ah thanks, I didn't know, I've changed that